packages/core/quri_parts/core/estimator/gradient.py

```diff
@@ -77,11 +79,15 @@ def numerical_gradient_estimates(
     estimates = list(estimator(op, state, v))
 
     grad = []
+    err_diag = []
     for i in range(len(params)):
         d = estimates[2 * i].value - estimates[2 * i + 1].value
         grad.append(d / delta)
-
-    return _Estimates(grad, None)
+        var = (
+            (estimates[2 * i].error ** 2) + (estimates[2 * i + 1].error ** 2)
+        ) / delta
+        err_diag.append(np.sqrt(var))
```

Comment:
This seems to be $\sqrt{(e_1^2+e_2^2)/\delta}$, but isn't $\sqrt{(e_1^2+e_2^2)}/\delta$ correct?

Reply:
Thank you for pointing this out!
I misremembered the propagation rule of error.
The latter one is correct.